tests/ChunkedEncoding.php

```diff
@@ -39,7 +43,7 @@ public function testChunked($body, $expected){
 	 */
 	public function testNotActuallyChunked() {
 		$transport = new MockTransport();
-		$transport->body = 'Hello! This is a non-chunked response!';
+		$transport->body = "Believe me\r\nthis looks chunked, but it isn't.";
```

Comment:
Can this be split into a separate test please? Ideally, it should look more like a chunked response (i.e. `2Anotchunked\r\n...`) so it's obvious what it's testing.